content/en/docs/aws/notebook-server.md

```diff
@@ -0,0 +1,25 @@
++++
+title = "Kubeflow Notebook Server on AWS"
+description = "Customize Kubeflow Notebook Server to use AWS Services"
+weight = 90
++++
+
+## Kubeflow Notebook Image
+
+Currently, we add ECR images on Kubeflow Notebook Image Options by-default. The list of ECR images we provide:
```

Comment:
You also need to mention this is built on top of DLC which give best DL framework performance on AWS

Reply:
Good point, added